genstudio-mlr-claims-app/src/genstudiopem/web-src/src/components/AdditionalContextDialog.tsx

```diff
@@ -103,7 +103,7 @@ export default function AdditionalContextDialog(): JSX.Element {
             Cancel
           </Button>
           <Button variant="primary" style="fill" onPress={handleClaimSelect}>
-            Select
+            Ok
```

Comment:
nit picking.
I think the correct spelling is either "OK" with all uppercase, or "Okay".